src/Nri/Ui/Checkbox/V5.elm

```diff
@@ -300,9 +300,8 @@ viewIcon styles icon =
         [ css
             [ position absolute
             , left zero
+            , paddingTop (px 4)
             , top (calc (pct 50) minus (px 18))
-            , border3 (px 2) solid transparent
```

Comment:
These styles were to account for the faked focus outline for keyboard users. Without the padding being even and without the transparent border, the focus styles looks kinda funky:

![image](https://user-images.githubusercontent.com/8811312/132594982-72de88aa-2e96-4cfb-ada6-933ec21dd333.png)